analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/ParseOptionsHelper.cs

```diff
@@ -94,7 +94,7 @@ language switch
             };
 
         private static ImmutableArray<ParseOptions> FilterByEnvironment(this IEnumerable<ParseOptions> options) =>
-            TestContextHelper.IsAzureDevOpsContext
+            TestContextHelper.IsNotPullRequestBuild
```

Comment:
When reading this, I miss the `local build uses single version as well` part. Wasn't the `IsAzureDevOpsContext` meant to be preserved here?